tests/test.py

```diff
@@ -464,32 +488,21 @@ def drop_tables():
     metadata.drop_all(engine_raw)
 
 
+# Set this to False to not rebuild binaries on setup.
+CLEAN_BINARIES = bool(os.environ.get('TEST_CLEAN_BINARIES', True))
+# Set this to False to not build binaries in principle.
+BUILD_BINARIES = True
```

Comment:
why not allow to set value from env too? as I understand, it's for manual debugging only?

Reply:
> why not allow to set value from env too? as I understand, it's for manual debugging only?

Yes, I did not add an environment variable for skipping builds because I did not think it was useful for manual debugging. It makes sense to build binaries and not clean them to avoid rebuilding, but it does not make sense – when working on Acra CE – to avoid building Acra CE binaries entirely. This hack is only necessary to run Acra CE tests with Acra EE binaries, where we want Acra CE tests to not build their binaries and use Acra EE ones.

Reply:
While I've been experimenting with `TEST_CLEAN_BINARIES` I have realized that `bool(os.environ.get(...))` does not really work as expected. Setting it to `0` or `false` does not parse into boolean `False`. In fact, it looks at whether the variable is set and whether it's empty. I have been setting and unsetting the variable for testing so I did not notice.

The same approach is used to get the value of TEST_MYSQL variable which I reused. Well, this is not nice, so I have added a utility to get boolean variables in a more convenient way:

```python
CLEAN_BINARIES = utls.get_bool_env('TEST_CLEAN_BINARIES', default=True)
```

Now `0`, `false`, `no`, `n`, and empty string are treated as `False`, while `1`, `true`, `yes,` and `y` are treated as `True`.